### ISSUE TITLE
PSA: upcoming OSRM version is going to announce via locations

### ISSUE THREAD

**Issue by DennisOSRM** [mapbox/mapbox-directions.js#41]
Instruction 9 has been implemented, e.g.:

``` JSON
        [
            "9",
            "Ravenéstraße",
            41,
            3,
            32,
            "40m",
            "W",
            267
        ]
```

as in 

``` JSON
{
    "status": 0,
    "route_instructions": [
        [
            "10",
            "Schönwalder Straße",
            61,
            0,
            24,
            "60m",
            "E",
            79
        ],
        [
            "7",
            "Kunkelstraße",
            96,
            1,
            14,
            "95m",
            "N",
            358
        ],
        [
            "7",
            "Ravenéstraße",
            179,
            2,
            26,
            "178m",
            "W",
            265
        ],
        [
            "9",
            "Ravenéstraße",
            41,
            3,
            32,
            "40m",
            "W",
            267
        ],
        [
            "3",
            "B 96",
            201,
            4,
            22,
            "201m",
            "N",
            356
        ],
        [
            "7",
            "Gerichtstraße",
            15,
            10,
            3,
            "15m",
            "W",
            290
        ],
        [
            "7",
            "B 96",
            115,
            11,
            1,
            "115m",
            "SW",
            204
        ],
        [
            "15",
            "",
            0,
            15,
            0,
            "0m",
            "N",
            0
        ]
    ],
    "route_name": [
        "Ravenéstraße",
        "B 96"
    ],
    "status_message": "Found route between points",
    "via_indices": [
        0,
        4,
        16
    ],
    "route_geometry": "gf|ecBwtdoXqEiv@yt@fBjGjcDh@nd@_WzAoD@iPDaPaEgUqGkWgNcBfLbf@b[jHaArEU|DA",
    "route_summary": {
        "total_distance": 708,
        "total_time": 93,
        "start_point": "Schönwalder Straße",
        "end_point": "B 96"
    },
    "via_points": [
        [
            52.542068,
            13.372252
        ],
        [
            52.5429,
            13.370455
        ],
        [
            52.54372,
            13.369666
        ]
    ],
    "found_alternative": false,
    "hint_data": {
        "checksum": 3564455985,
        "locations": [
            "Kg8BAMN0AgAfGAAAmwAAAAMAAAAAAAAACQAAACQZAAB0uiEDXAvMAAAAAAA",
            "2D4AANw-AABWBAAAPQAAAAsBAAAAAAAAAAAAAP____-0vSEDVwTMAAAAAAA",
            "ygkBAP____8WAAAADQAAABAAAABgAAAAEwAAAA-GAADowCEDQgHMAAQAAAA"
        ]
    }
}
```

![bildschirmfoto 2014-08-11 um 19 38 01](https://cloud.githubusercontent.com/assets/1067895/3880384/18046f1c-2183-11e4-8005-cd2dde87d15e.png)

cc: @jfirebaugh

**Comment by jfirebaugh** [comment]
Thanks for the heads up. What's the version number that introduces this?

**Comment by DennisOSRM** [comment]
Depends on how soon node-osrm build is fixed. Either 0.4.3 which is ready or 0.4.4

**Comment by DennisOSRM** [comment]
ref: https://github.com/Project-OSRM/osrm-backend/pull/1150

**Comment by DennisOSRM** [comment]
@jfirebaugh This is now in develop branch and will be available in the upcoming 0.4.3 release scheduled for tomorrow/Thursday.

**Comment by DennisOSRM** [comment]
@jfirebaugh this has been released with osrm v0.4.3 and the corresponding node-bindings v0.11.3.
